src/SharedPipelineComponents/DataLockEvents/SFA.DAS.Provider.Events.DataLock/Infrastructure/Data/DcfsDataLockEventPeriodRepository.cs

```diff
@@ -17,7 +17,8 @@ public class DcfsDataLockEventPeriodRepository : DcfsRepository, IDataLockEventP
                                        "CollectionPeriodYear," +
```

Comment:
one for later - should be refactored to a stored proc passing in a UDTT to reduce complexity and an uncached execution plan.